sdk/confidentialledger/confidential-ledger-rest/tests.yml

```diff
@@ -0,0 +1,15 @@
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/confidential-ledger"
+      ServiceDirectory: confidentialledger
+      EnvVars:
+        LEDGERURI: https://$(LEDGER_NAME).confidential-ledger.azure.com
```

Comment:
Only the `AZURE` prefixed env vars are necessary, as the bicep template outputs will set the others into the environment, provided you also move `LEDGER_IDENTITY` into the bicep outputs.